packages/bsky/src/index.ts

```diff
@@ -2,90 +2,54 @@
 // this is a relatively non-invasive change to express
 // they get handled in the error.handler middleware
 // leave at top of file before importing Routes
-import 'express-async-errors'
+import 'express-async-errors' // @TODO(bsky) remove
```

Comment:
oh?

I guess XRPC is pretty good at catching errors 🤔

Reply:
Yeah, I think we're pretty well taken care-of tbh!